src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadocmethod/InputMissingJavadocMethodInterfacePrivateMethod.java

```diff
@@ -3,6 +3,9 @@
 
 package com.puppycrawl.tools.checkstyle.checks.javadoc.missingjavadocmethod;
 
+/* Config:
+ * scope = public
```

Comment:
I do not know if we have reached a consensus on this yet, but I always enclose check  properties in `"`, i.e. `scope = "public"`.  One day in the future we will derive configs for UT's  from these comments, and some delimiter will help with that. (This goes for all below configs, too.)

Reply:
This seems like a good idea, I have made the changes.

Reply:
All except this one :)

Reply:
Done.

Reply:
Resolved.